src/devices/video/ef9365.cpp

```diff
@@ -1156,23 +1096,23 @@ void ef9365_device::ef9365_exec(uint8_t cmd)
 
 uint32_t ef9365_device::screen_update(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect)
 {
-	for(int j=0;j<bitplane_yres;j++)
+	for (int j = 0; j < bitplane_yres; j++)
 	{
-		for(int i=0;i<bitplane_xres;i++)
+		for (int i = 0; i < bitplane_xres; i++)
 		{
-			unsigned char color_index = 0x00;
+			uint8_t color_index = 0x00;
 
-			int ptr = ( bitplane_xres * j ) + i;
+			int ptr = (bitplane_xres * j) + i;
 
-			for(int p = 0; p < nb_of_bitplanes; p++)
+			for (int p = 0; p < nb_of_bitplanes; p++)
 			{
-				if( m_videoram->read_byte( (BITPLANE_MAX_SIZE*p) + (ptr>>3)) & (0x80>>(ptr&7)))
+				if (BIT(m_videoram->read_byte(BITPLANE_MAX_SIZE * p + (ptr >> 3)), 7 - (ptr & 7)))
```

Comment:
`7 - (ptr & 7)` can be simplified to `~ptr & 7` here.

Reply:
Applied.